org.jacoco.doc/docroot/doc/maven.html

```diff
@@ -41,11 +41,26 @@
   the tests with the <tt>javaagent</tt> set and no coverage would be recorded.
 </p>
 
+<p class="hint">
+  When using the <tt>maven-surefire-plugin</tt> with custom <a href="http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#argLine">
+   <tt>argLine</tt></a> you need to append <tt>@{argLine}</tt> to the custom <tt>argLine</tt><br>
+   <tt>&lt;argLine&gt;-Xms1024m -Xmx2048m <b>@{argLine}</b>&lt;/argLine&gt;</tt>
+</p>
+
+<p class="hint">
+  For a multi-module maven project using <tt>maven-surefire-plugin</tt> with custom <a href="http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#argLine">
+  <tt>argLine</tt></a> and not running JaCoCo runtime agent in all modules, you can set an empty <tt>argLine</tt> property.
+  This will prevent the build from failing with message <tt>Error: Could not find or load main class @{argLine}</tt>.<br>
```

Comment:
Currently build fails, because of unpaired `<br>`, which is invalid XHTML and should be `<br/>`. Also I don't think that we use `<br/>`, at least not inside of manually written files.